packages/legend-studio/src/models/protocols/pure/v1/transformation/pureGraph/from/V1_ConnectionTransformer.ts

```diff
@@ -229,8 +229,7 @@ const transformModelChainConnection = (
   element: ModelChainConnection,
 ): V1_ModelChainConnection => {
   const connection = new V1_ModelChainConnection();
-  // to make this consistent with grammar parser, ModelStore is a pseudo-store and will be skipped
-  connection.store = undefined;
+  connection.store = undefined; // @MARKER: GRAMMAR ROUNDTRIP --- omit this information during protocol transformation as it can be interpreted while building the graph
```

Comment:
Did we remove the ModelStore check in our own compiler

Reply:
no, this is just for transformer, the compiler check is still there